File: ΜΑΘΗΜΑ_9/UML-JAVA/employee.java

class employee
{
    private String name;
    private int age;
    private double salary;
    public employee(String n,int a,double s)
    {
        this.name=n;
        this.age=a;
        this.salary=s;
    }
    public String getName() {return this.name;}
    public int getAge() {return this.age;}
    public double getSalary() {return this.salary;}
    public void info()
    {
        System.out.println(this.name+"-"+String.valueOf(this.age)+"-"+String.valueOf(this.salary));
    }
}

File: 2020-2021/ΜΑΘΗΜΑ_9/UML-C++/employee.cpp
#include <iostream>

class employee
{
    std::string name;
    int age;
    double salary;
    public:
      employee(std::string n,int a,double s):name(n),age(a),salary(s) 
      {}
      std::string getName()const {return this->name;}
      int getAge()const {return this->age;}
      double getSalary()const {return this->salary;}
      void info()
      {
         std::cout<<this->name<<"-"<<this->age<<"-"
          <<this->salary<<std::endl;
      }
      friend std::ostream &operator<<(std::ostream &os,const employee &e)
      {
          return os<<e.name<<"-"<<e.age<<"-"<<e.salary<<std::endl;
      }
};